lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -321,13 +321,14 @@ newStakePoolLayer tr getEpCst db@DBLayer{..} nl metadataDir = StakePoolLayer
     (block0, _) = staticBlockchainParameters nl
 
     combineWith
-        :: (Quantity "lovelace" Word64 -> EpochConstants)
-        -> ([(StakePool, [PoolOwner])] -> IO [(StakePool, [PoolOwner])])
+        :: ([(StakePool, [PoolOwner])] -> IO [(StakePool, [PoolOwner])])
+        -> (Quantity "lovelace" Word64 -> EpochConstants)
```

Comment:
Diff here and above is just changing the order of the arguments to `combineWith`. Not needed, but think the sorting argument were originally meant to be the first argument, after "With".

Reply:
Side-note: this comment **should** really be in your commit message. Message of commits tell about the why. The commit itself tells the what :+1:

Reply:
I should make it a habit to review the diffs locally first, where I can easily edit the commits, but the GH-diff is convenient.

Reply:
You can always easily edit the commits after opening a PR you know :)

Reply:
I agree that the GH view is convenient. I often do the same.

Reply:
> You can always easily edit the commits after opening a PR you know :)
 
Yes, but on GH you can comment without "breaking the flow". Moving to your editor, finding the relevant source or commit is a context switch, especially when you haven't made it a habit or have setup your tools in a good way.